resources/views/graph.blade.php

```diff
@@ -0,0 +1,124 @@
+{{--This is the result page for the website of the filled in workflows. The result page shows the result of the Evidencio API call.
+This is represented in a chart, the user is able to choose the result to be diaplayed in different graphical chats by clicking on different buttons.
+The will be also a text result if any was included in the Evidencio API call.
+Users are also able to download a pdf of the results--}}
+
+
+{{--post request to Evidencio API with the answers of the workflow page. 
+This will return an array with the result and their parameters.--}}
+<?php
+use App\EvidencioAPI;
+if (!empty($_POST['answer'])&&!empty($_POST['model'])) {
+  $decodeRes = EvidencioAPI::run($_POST['model'],$_POST['answer']);
+}
+?>
+@extends('layouts.app')
+
+@section('content')
+@if(!empty($decodeRes['result']))
+<?php
+
+/**
+ * making data array for charts
+ *result is only one presentage value
+ *therefore creating opposite result
+ */
+$result=(int)$decodeRes['result'];
+$dataPoints = array(
+  array("label"=>"", "y"=>(100-$result)),
+  array("label"=>$decodeRes['title'], "y"=>$result),
+)
+?>
+
+{{--buttons for changing to the different chart representations--}}
+<div class="container">
+  <button id="pieButton" class="btn btn-primary btn-sm" value='pie' onclick="changeGraphType(this.value)">Pie Chart</button>
+  <button id="barButton" class="btn btn-primary btn-sm" value='bar' onclick="changeGraphType(this.value)">Bar Chart</button>
+  <button id="barButton" class="btn btn-primary btn-sm" value='doughnut' onclick="changeGraphType(this.value)">Doughnut Chart</button>
+  <button id="barButton" class="btn btn-primary btn-sm" value='polarArea' onclick="changeGraphType(this.value)">Polar Area Chart</button>
+</div>
+
+{{--Canvas for where the chart will be displayed--}}
+<div class="container">
+  <canvas id="graph"></canvas>
+</div>
+
+{{--Javascript for the creating the chat using Chartjs--}}
+<script>
```

Comment:
Reference a file instead of doing inline javascript for readability

Reply:
I would rather not create a external javascript file just for the chart, as I am using php variables for the data in the charts, I will also need to work with Ajax requests. Having to constantly make http requests when choosing different charts would slow it down a lot. The javascript is also quite short.